src/components/LoadingSpinner.tsx

```diff
@@ -1,73 +1,50 @@
-import { darkText, red500 } from "@/constants";
-import type { CSSProperties } from "react";
-import styled, { keyframes } from "styled-components";
+import { addOpacityToColor } from "@/helpers";
```

Comment:
I used this loading spinner library in the tools dapp and it looks and works much better than my custom one. I added it here because the existing loading spinner was looking odd after installing tailwind for reasons I can't figure out